pyproject.toml

```diff
@@ -49,7 +49,7 @@ invenio-admin = "==1.1.2"
 invenio-assets = "1.1.3"
 invenio-base = "==1.2.0"
 invenio-cache = "==1.0.0"
-invenio-celery = "==1.0.0"
+invenio-celery = "==1.1.0"
```

Comment:
why not ``1.1.1``?

Reply:
1.1.1 Locks celery on 4.2.x https://github.com/inveniosoftware/invenio-celery/commit/593b6647a948cf081b8efb42272f8613b9e98ef4

Reply:
:(